geode-protobuf/src/main/java/org/apache/geode/internal/protocol/protobuf/security/ProtobufSimpleAuthenticator.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.internal.protocol.protobuf.security;
+
+import static org.apache.geode.internal.protocol.protobuf.ProtocolErrorCode.AUTHENTICATION_FAILED;
+
+import org.apache.geode.internal.protocol.protobuf.AuthenticationAPI;
+
+import org.apache.geode.internal.protocol.protobuf.BasicTypes;
+import org.apache.geode.internal.protocol.protobuf.ClientProtocol;
+import org.apache.geode.internal.security.SecurityService;
+import org.apache.geode.security.AuthenticationRequiredException;
+import org.apache.geode.security.AuthenticationFailedException;
+
+import java.io.EOFException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Properties;
+
+import org.apache.shiro.subject.Subject;
+
+public class ProtobufSimpleAuthenticator implements Authenticator {
```

Comment:
Given this is a very specific authenticator, aka Protobuf for Shiro, could we not rename it to `ProtobufShiroAuthenticator`?

Reply:
Not a fan of the name as it's assuming a bit much about the behind the scenes implementation.  Really we're just calling into the securityService login and should care what it's doing.  Shiro isn't even necessarily the endpoint of the call, as it will dispatch the authentication call to a user provided security manager if one is available.

Reply:
The simple authentication is also misleading, as it is really a property based authenticator now.